src/org/thoughtcrime/securesms/recipients/Recipient.java

```diff
@@ -125,7 +125,8 @@ else if(dcChat!=null) {
         DcContext dcContext = DcHelper.getContext(context);
         int[] contacts = dcContext.getChatContacts(chatId);
         if( contacts.length>=1 ) {
-          maybeSetSystemContactPhoto(context, dcContext.getContact(contacts[0]));
+          dcContact = dcContext.getContact(contacts[0]);
+          maybeSetSystemContactPhoto(context, dcContact);
```

Comment:
```suggestion
          dcContact = dcContext.getContact(contacts[0]);
          maybeSetSystemContactPhoto(context, dcContact);
```

we want to store to the class, not overwrite the local variable.

Reply:
yikes! there was a local variable :sweat_smile: fixed